ckanext/switzerland/helpers.py

```diff
@@ -278,7 +278,7 @@ def resource_display_name(resource_dict):
     name = resource_dict.get("name", None)
     description = resource_dict.get("description", None)
     if name:
-        name = parse_json(name)
+        name = parse_json(eval(name))
```

Comment:
do we really need to use eval()  for json parsing?
or do we expect to get here some mathematical expressions?